packages/terra-form-input/src/terra-dev-site/doc/example/NumberInputField.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import InputField from '../../../InputField';
+
+const NumberInputField = () => (
+  <InputField
+    inputId="numeric-input"
+    label="Numeric Value"
+    help="Note: This can not be changed in the future"
```

Comment:
I don't think that this help text makes sense for this example.

Reply:
[removed](https://github.com/cerner/terra-core/blob/8bdcebd9da82c7a46dd80a17f6c6b5a98fd38049/packages/terra-form-input/src/terra-dev-site/doc/example/NumberInputField.jsx)